### PR TITLE
x11-terms/kitty: fix live ebuild and update python compatibility

### PR DESCRIPTION
**Pull request by end222**
Signed-off-by: Pablo Orduna <pabloorduna98@gmail.com>

Closes: https://bugs.gentoo.org/789285

### CODE REVIEW

**Review by juippis — COMMENTED** [review]
```
>>> Preparing source in /var/tmp/portage/x11-terms/kitty-9999/work/kitty-9999 ...
 * Applying kitty-9999-flags.patch ...
patching file setup.py
Hunk #4 FAILED at 789.
Hunk #5 succeeded at 804 (offset 1 line).
1 out of 5 hunks FAILED -- saving rejects to file setup.py.rej
 [ !! ]
 * ERROR: x11-terms/kitty-9999::gentoo failed (prepare phase):
 *   patch -p1  failed with /var/tmp/portage/x11-terms/kitty-9999/files/kitty-9999-flags.patch
```

**Comment by end222** [comment]
I see, the latest commit on kitty's repository (https://github.com/kovidgoyal/kitty/commit/d99e243f5717f5e3dcad1a7abef722f5d0609b3c) has messed up the patches.
I will fix it as soon as possible.

File: x11-terms/kitty/kitty-9999.ebuild [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7,8} )
+PYTHON_COMPAT=( python3_{7,8,9} )
```
Comment (thesamesam) [inline x11-terms/kitty/kitty-9999.ebuild:6]: Do any released versions in tree support 3.9?

**Comment by end222** [comment]
> ```
> >>> Preparing source in /var/tmp/portage/x11-terms/kitty-9999/work/kitty-9999 ...
>  * Applying kitty-9999-flags.patch ...
> patching file setup.py
> Hunk #4 FAILED at 789.
> Hunk #5 succeeded at 804 (offset 1 line).
> 1 out of 5 hunks FAILED -- saving rejects to file setup.py.rej
>  [ !! ]
>  * ERROR: x11-terms/kitty-9999::gentoo failed (prepare phase):
>  *   patch -p1  failed with /var/tmp/portage/x11-terms/kitty-9999/files/kitty-9999-flags.patch
> ```

Fixed!

File: x11-terms/kitty/kitty-9999.ebuild (hunk 1 above)
Comment (end222) [reply x11-terms/kitty/kitty-9999.ebuild:6]: Yes, the ebuild for the 0.20.3 version of kitty supports python 3.9

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**